src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -47,33 +47,38 @@
      */
     public function load(string $path, string ...$paths): void
     {
-        array_unshift($paths, $path);
-
-        foreach ($paths as $path) {
-            if (!is_readable($path) || is_dir($path)) {
-                throw new PathException($path);
-            }
+        $this->doLoad(false, $path, ...$paths);
```

Comment:
instead of creating a PHP array (with the variadic argument) to spread it again (here) to create another PHP array (variadic argument in the private method), it might make sense to pass an array to the private method instead of making it variadic.

Reply:
Indeed, we don't need a variadic argument in the private method.

Reply:
@stof & @fabpot new PR #28359 created :)